language-support/java/codegen/src/main/scala/com/digitalasset/daml/lf/codegen/CodeGenRunner.scala

```diff
@@ -118,15 +190,18 @@ object CodeGenRunner extends StrictLogging {
       s"Start processing packageIds '${interfaces.map(_.packageId).mkString(", ")}' in directory '${conf.outputDirectory}'"
     )
 
+    val prefixes = resolvePackagePrefixes(pkgPrefixes, conf.modulePrefixes, interfaces)
+    detectModuleCollisions(prefixes, interfaces)
+
     // TODO (mp): pre-processing and escaping
```

Comment:
The reason why I don't like initials beside `TODO`s is that `git blame` works better and initials tend to become irrelevant over time.

Reply:
I find them useful pretty frequently. Yes `git blame` works but it’s an extra step, it’s annoying to track it across refactorings and so on.